ci/matrix.xml

```diff
@@ -158,6 +173,13 @@
         'set scalaJSStage in Global := FullOptStage' \
         'set scalaJSOptimizerOptions in testSuite ~= (_.withDisableOptimizer(true))' \
         ++$scala testSuite/test \
+        testSuite/clean &&
+    sbtretry 'set inScope(ThisScope in testSuite)(jsEnv := SeleniumJSEnv(SeleniumFirefox).value)' \
+        ++$scala testSuite/test \
+        testSuite/clean &&
+    sbtretry 'set inScope(ThisScope in testSuite)(jsEnv := SeleniumJSEnv(SeleniumFirefox).value)' \
+        'set scalaJSStage in Global := FullOptStage' \
+        ++$scala testSuite/test \
```

Comment:
These changes should probably not be there. You're in the ES6 section, but they don't test ES6. These two can be removed, I think, they're duplicates of the ones in the ES5 section anyway.